### ISSUE TITLE
CMake fails boost regex alias not found

### ISSUE THREAD

**Issue by ferdymercury** [lamyj/odil#77]
On Linux, building with CMake fails after the latest commits from September:

```
CMake Error at src/CMakeLists.txt:22 (add_library):
  Target "libodil" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "move" links to target "Boost::regex" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "get" links to target "Boost::regex" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "genericscp" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "find" links to target "Boost::regex" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "qido-rs-client" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "dump" links to target "Boost::regex" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "stow-rs-client" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "store" links to target "Boost::regex" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "dicomdir" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at examples/cpp/CMakeLists.txt:23 (add_executable):
  Target "wado-rs-client" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_xml_converter" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_multipart_related" links to target "Boost::regex"
  but the target was not found.  Perhaps a find_package() call is missing for
  an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_URL" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_STOWRSResponse" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_DataSet" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_json_converter" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_EchoSCP" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_PresentationContextAC" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_STOWRSRequest" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CMoveResponse" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_dul_Transport" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_WADORSResponse" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_base64" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Writer" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Exception" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_MoveSCP" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_AssociationParameters" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_VRFinder" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_VR" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_SCU" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Reader" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_conversion" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_FindSCP" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Value" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_ImplementationClassUID" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_MoveSCU" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_UserIdentityAC" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_QIDORSResponse" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_StoreSCU" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_NCreateSCP" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Association" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_endian" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_FindSCU" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AAssociateAC" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_ElementAccessor" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CFindRequest" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_EchoSCU" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_GetSCP" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Tag" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CMoveRequest" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_BasicDirectoryCreator" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_UserInformation" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_Element" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CEchoRequest" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_GetSCU" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CFindResponse" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CGetResponse" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_MaximumLength" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CStoreRequest" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CStoreResponse" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_Message" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_RoleSelection" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CEchoResponse" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_Request" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_Response" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_UserIdentityRQ" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_Cancellation" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AAbort" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_dul_StateMachine" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_DcmtkException" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AAssociateRJ" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AReleaseRQ" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_WADORSRequest" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AAssociateRQ" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_HTTPResponse" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AReleaseRP" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_AsynchronousOperationsWindow" links to target
  "Boost::regex" but the target was not found.  Perhaps a find_package() call
  is missing for an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_uid" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_ApplicationContext" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_ImplementationVersionName" links to target "Boost::regex"
  but the target was not found.  Perhaps a find_package() call is missing for
  an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_Item" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_SCPDispatcher" links to target "Boost::regex" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_PDataTF" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_HTTPRequest" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_message_CGetRequest" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_PresentationContextRQ" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_AssociationAcceptor" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_SOPClassCommonExtendedNegotiation" links to target
  "Boost::regex" but the target was not found.  Perhaps a find_package() call
  is missing for an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_registry" links to target "Boost::regex" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_ItemWithParameters" links to target "Boost::regex"
  but the target was not found.  Perhaps a find_package() call is missing for
  an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_Message" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_Selector" links to target "Boost::regex" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_unicode" links to target "Boost::regex" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_pdu_SOPClassExtendedNegotiation" links to target
  "Boost::regex" but the target was not found.  Perhaps a find_package() call
  is missing for an IMPORTED target, or an ALIAS target is missing?


CMake Error at tests/CMakeLists.txt:34 (add_executable):
  Target "test_webservices_QIDORSRequest" links to target "Boost::regex" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
```

**Comment by ferdymercury** [comment]
Ok, I saw now one warning above:

```
-- Found Boost: /usr/include (found version "1.65.1") found components: exception filesystem log unit_test_framework system date_time log_setup thread chrono atomic missing components: regex
-- Configuring done
```

Maybe it should give an error rather than a warning?

Installing libboost-regex-dev solves it.

**Comment by ferdymercury** [comment]
Maybe it could be added here:
https://github.com/lamyj/odil/blob/master/OdilConfig.cmake.in#L5

**Comment by lamyj** [comment]
This is unusual as Odil does not use Boost.Regex (`grep -ri regex src/ wrappers/ tests/`). Could you give me a bit more details about your environment (OS, version of Boost, JsonCpp, ICU and if applicable DCMTK)?

**Comment by ferdymercury** [comment]
Sure!
Thanks for the reply.

```
-- Found Boost: /usr/include (found version "1.65.1") found components: date_time exception filesystem log system log_setup thread regex chrono atomic 
-- Found the following ICU libraries:
--   uc (required)
-- Found ICU: /usr/include (found version "60.2") 
-- Trying to find DCMTK expecting DCMTKConfig.cmake
-- Trying to find DCMTK expecting DCMTKConfig.cmake - ok
-- Found Boost: /usr/include (found version "1.65.1")  
-- Trying to find DCMTK expecting DCMTKConfig.cmake
-- Trying to find DCMTK expecting DCMTKConfig.cmake - ok
-- Found Boost: /usr/include (found version "1.65.1") found components: exception filesystem log unit_test_framework system date_time log_setup thread regex chrono atomic 
-- Trying to find DCMTK expecting DCMTKConfig.cmake
-- Trying to find DCMTK expecting DCMTKConfig.cmake - ok
-- Trying to find DCMTK expecting DCMTKConfig.cmake
-- Trying to find DCMTK expecting DCMTKConfig.cmake - ok
-- Configuring done
-- Generating done
-- Build files have been written to: /opt/odil/build
```
`libdcmtk-dev is already the newest version (3.6.2-3build3).`

```
Linux 5.4.0-47-generic #51~18.04.1-Ubuntu SMP Sat Sep 5 14:35:50 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=19.3
DISTRIB_CODENAME=tricia
DISTRIB_DESCRIPTION="Linux Mint 19.3 Tricia"
```

```
CMakeFiles/Export/_opt/odil/install/lib/CMake/Odil/OdilTargets.cmake:53:  INTERFACE_LINK_LIBRARIES "Boost::date_time;Boost::exception;Boost::filesystem;Boost::log;\$<\$<BOOL:ON>:ofstd;oflog;dcmdata;i2d;dcmimgle;dcmimage;dcmjpeg;ijg8;ijg12;ijg16;dcmjpls;dcmtls;dcmnet;dcmsr;cmr;dcmdsig;dcmwlm;dcmqrdb;dcmpstat;dcmrt;dcmiod;dcmfg;dcmseg;dcmtract;dcmpmap>;ICU::uc;JsonCpp::JsonCpp;\$<\$<PLATFORM_ID:Windows>:netapi32>;\$<\$<BOOL:>:bcrypt;iphlpapi>"
CMakeFiles/Makefile.cmake:45:  "/snap/cmake/599/share/cmake-3.18/Modules/FindBoost.cmake"
OdilConfig.cmake:5:find_dependency(Boost REQUIRED COMPONENTS date_time exception filesystem log)
OdilTargets.cmake:50:  INTERFACE_LINK_LIBRARIES "Boost::date_time;Boost::exception;Boost::filesystem;Boost::log;\$<\$<BOOL:ON>:ofstd;oflog;dcmdata;i2d;dcmimgle;dcmimage;dcmjpeg;ijg8;ijg12;ijg16;dcmjpls;dcmtls;dcmnet;dcmsr;cmr;dcmdsig;dcmwlm;dcmqrdb;dcmpstat;dcmrt;dcmiod;dcmfg;dcmseg;dcmtract;dcmpmap>;ICU::uc;JsonCpp::JsonCpp;\$<\$<PLATFORM_ID:Windows>:netapi32>;\$<\$<BOOL:>:bcrypt;iphlpapi>"
```

**Comment by lamyj** [comment]
Found it: Boost.log requires Boost.regex (source: `/usr/share/cmake-3.10/Modules/FindBoost.cmake` look for `if(NOT Boost_VERSION VERSION_LESS 106500)`) but the libboost-log-dev package does not depend on libboost-regex-dev. This is distribution-dependent, so I've just fixed the doc (cf. 35b51e2).

As a side note, the version of pybind11 packaged in Mint 19 is too old to compile the Python wrappers. I've updated the documentation to specify the minimum required versions of the dependencies (2f1844f).
